packages/components/tabs/src/tabs.tsx

```diff
@@ -120,6 +92,11 @@ export default defineComponent({
 
   setup(props, { emit, slots, expose }) {
     const instance = getCurrentInstance()!
+    const ns = useNamespace('tabs')
+
+    const nav$ = ref<TabNavInstance>()
+    const panes: Record<number, TabsPaneContext> = reactive({})
```

Comment:
Just for consistency, can we do:

```suggestion
const panes = reactive<Record<number, TabsPaneContext>>({})
```

This way we can keep the code consistent with line#97

Reply:
Updated.